ci-wheel.Dockerfile

```
@@ -164,38 +173,49 @@ RUN curl https://pyenv.run | bash
 # TODO: Determine if any cleanup of the pyenv layers is needed to shrink the container
 RUN pyenv update
 
-RUN case "${LINUX_VER}" in \
-    "ubuntu"*) \
-        pyenv install --verbose "${RAPIDS_PY_VERSION}" \
-      ;; \
-    "centos"*) \
-        # Need to specify the openssl location because of the install from source
-        CPPFLAGS="-I/usr/include/openssl" LDFLAGS="-L/usr/lib" pyenv install --verbose "${RAPIDS_PY_VERSION}" \
-      ;; \
-    "rockylinux"*) \
-        CPPFLAGS="-I/usr/include/openssl" LDFLAGS="-L/usr/lib" pyenv install --verbose "${RAPIDS_PY_VERSION}" \
-      ;; \
-    *) \
-      echo "Unsupported LINUX_VER: ${LINUX_VER}" && exit 1; \
-      ;; \
-  esac
-
-RUN pyenv global ${PYTHON_VER} && python -m pip install auditwheel patchelf twine rapids-dependency-file-generator dunamai && pyenv rehash
+RUN <<EOF
+case "${LINUX_VER}" in
+  "ubuntu"*) 
+    pyenv install --verbose "${RAPIDS_PY_VERSION}"
+    ;;
+  "centos"*) 
+    # Need to specify the openssl location because of the install from source
+    CPPFLAGS="-I/usr/include/openssl" LDFLAGS="-L/usr/lib" pyenv install --verbose "${RAPIDS_PY_VERSION}"
+    ;;
+  "rockylinux"*) 
+    CPPFLAGS="-I/usr/include/openssl" LDFLAGS="-L/usr/lib" pyenv install --verbose "${RAPIDS_PY_VERSION}"
+    ;;
+  *) 
+    echo "Unsupported LINUX_VER: ${LINUX_VER}" && exit 1
```

```suggestion
    echo "Unsupported LINUX_VER: ${LINUX_VER}"
    exit 1
```

Let's remove the `&&` since we're no longer using it anywhere else.